src/styles/app/05-main-area.css

```
@@ -468,148 +183,71 @@
 	color: var(--text-primary);
 }
 
-.welcomeTipBody {
+.welcomeActionHint {
 	font-size: var(--text-xs);
-	line-height: var(--leading-snug);
 	color: var(--text-secondary);
+	line-height: var(--leading-relaxed, 1.6);
+	word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)